flake.nix

```diff
@@ -353,10 +413,8 @@
             # https://flake.parts/options/flake-parts.html?highlight=perSystem.apps#opt-perSystem.apps
             apps = {
               holochain.program = "${holochain}/bin/holochain";
-              hc.program = "${holochain}/bin/hc";
-              hc-run-local-services.program = "${holochain}/bin/hc-run-local-services";
-              hc-sandbox.program = "${holochain}/bin/hc-sandbox";
-              hcterm.program = "${holochain}/bin/hcterm";
+              hc.program = "${hc}/bin/hc";
```

Comment:
More straight forward now that we can just reference their own packages and not take apart the Holochain one. But also had to remove `hc-run-local-services` that was still hanging around.